packages/block-library/src/separator/edit.js

```diff
@@ -44,10 +52,27 @@ export default function SeparatorEdit( { attributes, setAttributes } ) {
 		color: currentColor,
 		backgroundColor: currentColor,
 	};
+	const Wrapper = tagName === 'hr' ? HorizontalRule : tagName;
```

Comment:
Why not just use the `tagName` instead of `HorizontalRule` which is `hr`?

Reply:
HorizontalRule is something else in native app (I don't want to impact that for now)